---
layout: default
---
		<!-- Main -->
			<article id="main">
					
				<!-- One -->
					<section class="wrapper style4 container">
					
						<!-- Content -->
							<div class="content">
								<section>
										<header>
										<h3><strong>Miscellaneous projects
</strong></h3> </header>

<p>A place for all the projects that don't merit their own page (yet). Currently these include (1) metagenomic data from hot springs in Kamchatka, Russia, (2) metagenomic data from aquariums, (3) obligate insect-associated fungi, and (4) viral discovery efforts. </p>


<div class="row">

	<div class="6u">
		<br>
		<br>
		<a class="image featured special"><img src="images/Ark_RussellNeches.jpg" alt="Photo of Arkashin Shurf, Kamchatka, Russia taken by Russell Neches; https://www.flickr.com/photos/rneches/" /></a>
	</div>

<div class="6u">

<header> <h4><strong>(1) Metagenomic data from two thermal pools in Kamchatka, Russia</strong></h4></header>
<p>This project was a collaboration with <a href="https://www.ecoevoint.ch/">Dr. Laetitia Wilkins</a>. Together we assembled draft metagenome-assembled genomes (MAGs)  from two hot springs located in the Kamchatka peninsula in Russia. We recovered 36 total MAGs including members that were taxonomically assigned to groups previously underrepresented in available genome data. This included several archaea (Korarchaeota, Bathyarchaeota and Aciduliprofundum) and one potentially new species within the bacterial genus Sulfurihydrogenibium. </p>


</div>

</div>
		<p>
<b>Related publications:</b>
<ul class="default">
<li><b>Ettinger CL*</b>, Wilkins LGE*, Jospin G & Eisen JA. 2019. Metagenome-assembled genomes provide new insight into the microbial diversity of two thermal pools in Kamchatka, Russia. Scientific Reports. DOI: <a href=https://www.nature.com/articles/s41598-019-39576-6>10.1038/s41598-019-39576-6</a> *co-first authors</li>		
</ul>	
</p>

<header> <h4><strong>(2) Metagenomic data from aquariums</strong></h4></header>
<p>Previously, members of the Eisen lab sampled aquaria as part of an undergraduate research project that investigated microbial community assembly in the fall of 2012. As part of that work, metagenomes were generated from various aquaria. Later, using these metagenomes we were able to recover 11 MAGs reconstructed from freshwater and saltwater aquaria, including representatives of Polynucleobacter, Anaerolinea, Roseobacter, Flavobacteriia, Octadecabacter, Mycobacterium, and Candidate Phyla Radiation (CPR) members. </p>

		<p>
<b>Related publications:</b>
<ul class="default">
	
<li><b>Ettinger CL</b>, Bryan J*, Tokajian S, Jospin G, Coil D & Eisen JA. 2021. Reconstruction of metagenome-assembled genomes from aquaria. Microbial Resource Announcements. DOI:<a href="https://journals.asm.org/doi/10.1128/MRA.00557-21"> 10.1128/MRA.00557-21</a> *undergraduate author </li>

</ul>	
</p>
<p>
	
	<header> <h4><strong>(3) Obligate Insect-Associated Fungi </strong></h4></header>
	<p>I have been involved in several projects focused on the genomics of obligate insect-associated fungi. One project involved assisting with removing symbionts or contaminant bacterial genomes from the genome of Massospora cicadina an obligate fungal parasite of cicadas. Another project involved comparative genomics and transcriptomics across development of the blastoclad fungus, Coelomomyces lativittatus, an obligate parasite of mosquitoes and microcrustaceans.  </p>

		<p>
<b>Related publications:</b>
<ul class="default">

	<li><b>Ettinger CL</b>, Ostovar T, Yacoub M, Ahrendt S, Hice RH, Federici BA, & Stajich JE. 2023. Genomes and transcriptomes help unravel the complex life cycle of the blastoclad fungus, Coelomomyces lativittatus, an obligate parasite of mosquitoes and microcrustaceans. Mycologia. DOI: <a href="https://doi.org/10.1080/00275514.2023.2228182"> 10.1080/00275514.2023.2228182</a>  </li>
	
	<li>Stajich JE, Lovett BR, <b>Ettinger CL</b>, Carter-House D, Kurbessoian T & Kasson MT. 2022. An Improved 1.5 Gigabase Draft Assembly of Massospora cicadina (Zoopagomycota), Obligate Fungal Parasite of 13- and 17-Year Cicadas. Microbial Resource Announcements. DOI: <a href="https://doi.org/10.1128/mra.00367-22"> 10.1128/mra.00367-22</a>  </li>
		
	<li><b>Ettinger CL</b>, Lovett BR, Kasson MT & Stajich JE. 2022. Metagenome-assembled Genomes of Bacteria associated with Massospora cicadina Fungal Plugs from Infected Brood VIII Periodical Cicadas. Microbial Resource Announcements. DOI: <a href="https://doi.org/10.1128/mra.00413-22"> 10.1128/mra.00413-22</a>  </li>
	
</ul>	
</p>

	<p>
		
		<header> <h4><strong>(4) Viral discovery efforts</strong></h4></header>
		<p>Over the course of summer 2021 and 2022, I mentored remote-based undergraduate interns who worked on viral discovery projects from metagenomic data. These projects focused on viruses from terrestrial cyanobacteria metagenomes and Antarctic rock metagenomes.  </p>


		<p>
<b>Related publications:</b>
<ul class="default">

	<li><b>Ettinger CL</b>, Saunders M*, Selbmann L, Delgado-Baquerizo M, Donati C, Albanese D, Roux S, Tringe S, Pennacchio C, Stajich JE, & Coleine C. 2023. Highly diverse and unknown viruses may enhance Antarctic endolithsâ€™ adaptability. Microbiome. DOI: <a href="https://doi.org/10.1186/s40168-023-01554-6"> 10.1186/s40168-023-01554-6</a> *undergraduate author </li>

	<li><b>Ettinger CL</b>, Paul S*, Flores N*, Pietrasiak N, & Stajich JE. Submitted. Exploring Viral Communities Associated with Terrestrial Cyanobacteria Metagenomes. Preprint available on bioRxiv. DOI:<a href="https://doi.org/10.1101/2022.12.14.520320 "> 10.1101/2022.12.14.520320</a> *undergraduate author </li>

</ul>	
</p>
		
												
									
								
							
								</section>
							</div>

					</section>
					
					
					
					
						
										
								
					
			</article>
