---
layout: default
---
		<!-- Main -->
			<article id="main">
					
				<!-- One -->
					<section class="wrapper style4 container">
					
						<!-- Content -->
							<div class="content">
								<section>
										<header>
										<h3><strong>Environmental drivers of the seagrass microbiome
</strong></h3> </header>

<div class="row">
<div class="6u">
<a class="image featured special"><img src="images/seagrass.jpg" alt="Photo of seagrass" /></a></div>
<div class="6u">
<p> A series of projects investigating the potential biotic and abiotic drivers of microbiome assembly in seagrass beds including observing (1) differences across locations within a seagrass patch, (2) differences related to proximity to a marina and (3) succession during ammonification. These projects were associated with a larger effort, the <a href="https://seagrassmicrobiome.org/">Seagrass Microbiome Project</a>.</p>

</div></div>
									
									
									<header>
										<h4><strong>(1) Seagrass microbiome variation at patch edge</strong></h4>
									</header>
								
								<div class="row">
									<div class="6u">
										<a class="image featured special"><img src="images/edges.png" alt="Edge diagram" /></a>
									</div>
									
									<div class="6u">
										<p>Seagrass beds are progressively affected by climate change, pollution and habitat destruction. As a result, seagrass is vulnerable to habitat fragmentation. This fragmentation expands the perimeter of the seagrass beds while decreasing the area the beds occupy, significantly changing the habitat available to macrofauna. Habitat fragmentation can thus lead to greater edge effects and results in increased biodiversity at the edge of habitats. </p>
									</div>
								</div>
								<div class="row">
									<div class="6u"><p>
										The goal of this project was to compare microbial communities obtained from the <strong> inside</strong>, <strong> edge </strong> and <strong>outside </strong> of beds of the seagrass, Zostera marina. We found that microbial communities differed  between sample types (root, leaf and sediment) and in sediments from different sites (inside, edge, outside), but did not differ for seagrass tissues (leaf, root) between different sites (inside, edge). Carbon:Nitrogen ratio and eelgrass density were both correlated to sediment community composition, indicating a likely role for Z. marina in sediment community assembly. </p>
									</div>
									<div class="6u">
										<a class="image featured special">
											<img  src="images/Ettinger_PeerJ_2017a_Figure6.png" alt="Fig. 6 from Ettinger et al 2017" /> 
											<br>
											<p>Interrogating the relationship between environmental data and sediment microbial communities revealed correlation between seagrass density and C:N ratio with sediment from inside patches.</p>
										</a>
									</div>
									
								</div>
									
									<header>
											<h4><strong>(2) Effect of marina proximity on seagrass microbiome</strong></h4>
										</header>
					
										<p> Seagrass populations are susceptible to human impact, particularly in high traffic areas like marinas. Here, as part of a larger project looking at tunicate abundance led by Sofie Voerman, we investigated whether proximity to a marina is correlated with the taxonomic composition and structure of the Zostera marina microbiome. The RMarkdown file for the analysis can be found on <a href="https://figshare.com/articles/Marina_proximity_16S_rRNA_gene_amplicon_analysis/6998636"> Figshare. </a> Further discussion of this project is also available in <a href="https://www.proquest.com/dissertations-theses/taxonomic-diversity-bacterial-fungal-communities/docview/2460090049/se-2?accountid=201395"> Chapter 2 of my dissertation.</a>  </p>
										
										<header>
											<h4><strong>(3) Ammonification in seagrass sediments</strong></h4>
										</header>
										
										
										<div class=row>
											<div class="6u">
												<p> An important process that occurs in seagrass beds is ammonification, the first step in early diagenesis (the combination of biological, chemical and physical processes that act on deposited organic matter). However not much is known about how this process is affected by microbial community composition or seagrass bed characteristics. Here, we sequenced the 16S rRNA gene to determine the microbial community composition of seagrass bed sediments with the goal of identifying which member(s) of the community might be responsible for the ammonification rates observed. Sediment was collected from 72 plots from a 15 month long field experiment in which Zostera marina genotypic richness and relatedness were manipulated and the rate of ammonification was measured in sediment from these plots at several different time points post field collection. While we observed microbial community succession related to sulfur cycling, we found no correlation between the community and rates of ammonification. </p> 
											</div>
											<div class="6u">
												<a class="image featured special"><img src="images/Ettinger_PeerJ_2017b_Figure4.png" alt="Fig. 4 from Ettinger et al 2017" />
												<br>
												<p> The relative abundance of taxonomic orders varied over time. We observed a relative decrease in Desulfobacterales concomitant with an increase in Thiotrichales and Alteromonadales which we proposed was the result of the coupling of sulfate reduction and sulfide oxidation during the experiment.
</p>
												</a>
											</div>	
										</div>
								
								
								
								<p>
								<b>Related publications:</b>
								<ul class="default">
								<li><b>Ettinger CL</b>, Williams SL, Abbott JM, Stachowicz JJ & Eisen JA. 2017.  Microbiome succession during ammonification in eelgrass bed sediments. PeerJ. DOI:<a href="https://peerj.com/articles/3674/"> 10.7717/peerj.3674 </a> </li>

								<li><b>Ettinger CL</b>, Voerman SE, Lang JM, Stachowicz JJ & Eisen JA. 2017. Microbial communities in sediment from Zostera marina patches, but not the Z. marina leaf or root microbiomes, vary in relation to distance from patch edge. PeerJ. DOI:<a href="https://peerj.com/articles/3246/">10.7717/peerj.3246</a></li>		
								</ul>	
								</p>
										
										
										
									
							
								</section>
							</div>

					</section>
					
					
					
					
						
										
								
					
			</article>
